util/topsql/stmtstats/kv_exec_count.go

```diff
@@ -49,6 +50,9 @@ type KvExecCounter struct {
 // the TiKV dimension.
 func (c *KvExecCounter) RPCInterceptor() interceptor.RPCInterceptor {
 	return func(next interceptor.RPCInterceptorFunc) interceptor.RPCInterceptorFunc {
+		if !topsqlstate.TopSQLEnabled() {
```

Comment:
How if
```suggestion
                return func(target string, req *tikvrpc.Request) (*tikvrpc.Response, error) {
			if !topsqlstate.TopSQLEnabled() {
			    c.mark(target)
			}
			return next(target, req)
		}
```

Reply:
I prefer the original, since when topsql is disabled, there will one less function call.